.github/workflows/changeset-release.yml

```diff
@@ -49,3 +66,10 @@ jobs:
           GIT_AUTHOR_EMAIL: github-actions[bot]@users.noreply.github.com
           GIT_COMMITTER_NAME: github-actions[bot]
           GIT_COMMITTER_EMAIL: github-actions[bot]@users.noreply.github.com
+
+      - name: Publish to npm (if not already published)
+        if: steps.changesets_files.outputs.exists != 'true'
+        run: pnpm release
+        env:
+          GITHUB_TOKEN: ${{ secrets.IMJS_ADMIN_GH_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPMJS_PUBLISH_ITWIN }}
```

Comment:
ugh now it is failing auth, i thought providing `NPM_TOKEN` would be sufficient 🤦

https://github.com/iTwin/iTwinUI/actions/runs/8254825928/job/22579866651#step:9:148